rfcs/0000-rfc-session.md

```diff
@@ -0,0 +1,205 @@
+# 0000: Session
+<!--^ The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC, taking care not to conflict with other RFCs.-->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage -->
+- Date: 6/22/2020 <!-- Update to reflect date of most recent stage advancement -->
```

Comment:
Note for the future: we set this at the time we merge the RFC. No need to twiddle with it :-)

No action is needed, though.